src/main/java/org/mojohaus/mojo/wagon/AbstractCopyMojo.java

```diff
@@ -1,4 +1,4 @@
-package org.codehaus.mojo.wagon;
+package org.mojohaus.mojo.wagon;
```

Comment:
maybe package change in separate PR

Reply:
Local test which I wouldn't have wanted to commit. Thanks for catching!